lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -29,11 +31,12 @@ def find_df
     end
 
     def command
-      [command_path, "-B", "#{size}", *path]
+      ENV['LC_ALL'] = 'C'
```

Comment:
this actually change the ENV for the whole process? maybe you need to run in and return to the original env value? see https://github.com/puppetlabs/facter/blob/2.x/lib/facter/core/execution/base.rb for some examples.